src/map/skill.cpp

```diff
@@ -4310,7 +4311,6 @@ static TIMER_FUNC(skill_timerskill){
 					}
 					break;
 				case NPC_REVERBERATION_ATK:
-				case WM_REVERBERATION_MAGIC:
```

Comment:
Removes animation?

Reply:
Aegis doesn't seem to send this anymore. Probably related to the newer client needed to display the new animation.